core/primitives-core/Cargo.toml

```diff
@@ -19,12 +19,14 @@ bs58 = "0.4"
 derive_more = "0.99.3"
 num-rational = { version = "0.3.1", features = ["serde"]}
 serde = { version = "1", features = ["derive"] }
-serde_json = "1"
 sha2 = "0.9"
 deepsize = { version = "0.2.0", optional = true }
 
 near-account-id = { path = "../account-id" }
 
+[dev-dependencies]
+serde_json = "1"
```

Comment:
oh, that's excellent, it's a somewhat largish dep, and it's useful to move it to a dev-dep of a foundational crate!